packages/story-editor/src/components/checklist/checks/imageElementResolution.js

```diff
@@ -37,41 +37,20 @@ import { filterStoryElements, getVisibleThumbnails } from '../utils';
 import { useRegisterCheck } from '../countContext';
 import { useIsChecklistMounted } from '../popupMountedContext';
 
-export function mediaElementResolution(element) {
-  switch (element.type) {
-    case 'image':
-      return imageElementResolution(element);
-    case 'gif':
-      return gifElementResolution(element);
-    default:
-      return false;
-  }
-}
-
-function imageElementResolution(element) {
-  const heightResTooLow =
-    element.resource?.sizes?.full?.height < 2 * element.height;
-  const widthResTooLow =
-    element.resource?.sizes?.full?.width < 2 * element.width;
-
-  return heightResTooLow || widthResTooLow;
-}
-
-function gifElementResolution(element) {
-  // gif/output uses the MP4 video provided by the 3P Media API for displaying gifs
-  const heightResTooLow =
-    element.resource?.output?.sizes?.mp4?.full?.height < 2 * element.height;
-  const widthResTooLow =
-    element.resource?.output?.sizes?.mp4?.full?.width < 2 * element.width;
+// TODO: This does not seem to accommodate for scaling.
+export function imageElementResolution(element) {
+  const imageResolutionLow =
+    element.resource?.height < 2 * element.height ||
+    element.resource?.width < 2 * element.width;
 
-  return heightResTooLow || widthResTooLow;
+  return ['image', 'gif'].includes(element.type) && imageResolutionLow;
 }
```

Comment:
Nice fix! this works for videos for me now. Do we want this to trigger with gifs too @BrittanyIRL?

![image](https://user-images.githubusercontent.com/22185279/128914035-490302a5-9f24-43da-b038-c3426166fde2.png)

Reply:
I'm getting this check for gifs, which makes sense to me. i think we're good. 
<img width="597" alt="Screen Shot 2021-08-10 at 12 51 48 PM" src="https://user-images.githubusercontent.com/10720454/128925895-7257721d-190d-405f-adcb-2ab630c157ae.png">